device-types/Digi/Digi-Passport-16.yml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: Digi
+model: Digi Passport 16
+slug: digi-passport-16
+part_number: '70002260'
+u_height: 1
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (6.09 lbs or 2.78 kg)